api/serverpb/json/serverpb.json

```diff
@@ -386,17 +387,59 @@
                   "x-order": 3
                 },
                 "disable_stt": {
+                  "description": "Disable Security Threat Tool.",
                   "type": "boolean",
-                  "title": "Disable Security Threat Tool",
                   "x-order": 11
                 },
                 "disable_telemetry": {
                   "type": "boolean",
                   "x-order": 1
                 },
+                "email_alerting_settings": {
+                  "description": "EmailAlertingSettings represents email (SMTP) configuration for Integrated Alerting.",
+                  "type": "object",
+                  "properties": {
+                    "from": {
+                      "description": "SMTP From header field.",
+                      "type": "string",
+                      "x-order": 0
+                    },
+                    "hello": {
+                      "description": "Hostname to identify to the SMTP server.",
+                      "type": "string",
+                      "x-order": 2
+                    },
+                    "identity": {
+                      "description": "Auth using PLAIN.",
+                      "type": "string",
+                      "x-order": 5
+                    },
+                    "password": {
+                      "description": "Auth using LOGIN and PLAIN.",
+                      "type": "string",
+                      "x-order": 4
+                    },
+                    "secret": {
+                      "description": "Auth using CRAM-MD5.",
+                      "type": "string",
+                      "x-order": 6
+                    },
+                    "smarhost": {
```

Comment:
Is this a term or an abbreviation? Not a typo, I hope :)

Reply:
Term: https://en.wikipedia.org/wiki/Smart_host (Alertmanager calls it "smarthost", one word)

Reply:
lol, then it's a typo!

Reply:
Doh!

Reply:
Fixed